packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -84,6 +84,7 @@ describe('Launchpad: Setup Project', () => {
       cy.containsPath('cypress/support/e2e.js')
       cy.containsPath('cypress/support/commands.js')
       cy.containsPath('cypress/fixtures/example.json')
+      cy.get('[data-cy=collapsible-header]').invoke('attr', 'aria-expanded').should('eq', 'false')
```

Comment:
```suggestion
      cy.get('[data-cy=collapsible-header]').should('have.attr', 'aria-expanded', 'false')
```

I'm not a big fan of invoke when you can use something more standard. This comment applies to all other cases where invoke has been added.